src/app/login/components/Login.tsx

```diff
@@ -69,12 +70,24 @@ export const LoginPage = () => {
   } = form;
 
   return (
+    <FormWrapper>
     <div className="h-full flex items-center justify-center p-4">
-      <div className="bg-white rounded-lg shadow-xl overflow-hidden max-w-md w-full">
-        <div className="bg-gray-800 p-8 text-center">
-          <h2 className="text-2xl font-bold text-white">Welcome Back</h2>
-          <p className="text-gray-400">Log in to your Shoes Rack account</p>
-        </div>
+    <div className="bg-white rounded-lg shadow-2xl overflow-hidden max-w-md w-full">
+      <div className="flex justify-center item-center pt-8 ">
+        <Image
+          src={LogoImage}
+          width="65"
+          height="65"
+          alt={"logo"}
+          className="border border-black rounded-full p-1"
```

Comment:
@muhammad-bilal-ali-khokhar  why we need to add padding for an image?

Reply:
@nadirhussain786 if I not give padding, image will touch the border and UI will look not good